components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/provisioning/impl/DefaultProvisioningHandler.java

```diff
@@ -112,7 +112,8 @@ public void handle(List<String> roles, String subject, Map<String, String> attri
 
         RegistryService registryService = FrameworkServiceComponent.getRegistryService();
         RealmService realmService = FrameworkServiceComponent.getRealmService();
-
+        String syncMethod = IdentityUtil.threadLocalProperties.get()
```

Comment:
No null check required on get call?

Reply:
This get call will return the empty list if there are no elements. And sync method will be set before executing this 
https://github.com/wso2/carbon-identity-framework/pull/4434/files#diff-f4f72672a99b2740f6bd66c5635bfdd9e446d2628bbabf7ed410fad7014f096fR617

Reply:
This get call will return the empty list if there are no elements. And sync method will be set before executing this 
https://github.com/wso2/carbon-identity-framework/pull/4434/files#diff-f4f72672a99b2740f6bd66c5635bfdd9e446d2628bbabf7ed410fad7014f096fR617